### PR TITLE
Upgrade to Webtokens 2.0

### LINKED ISSUE #74: web-token 2?
Hi,

would you consider updating the dependency on `web-token` to version 2? It would mean having a min PHP version of 7.2, but according to #72 that seems to be on the agenda anyway...
I could give it a try and create a PR, if wanted.

### PR DESCRIPTION
**Pull request by rasmusbe**
This PR replaces #85 since it became stale and I couldn't write to it since I didn't own the head repo. Closes #74.

This PR includes:
- Changes to support Webtokens 2.0 (work done by @CatoTH)
- Upgraded PHPUnit to version 8, possible after #87 got accepted.
- Fixed style errors found by `composer check-style` since that wasn't done before

### CODE REVIEW

**Comment by rasmusbe** [comment]
Coverage only decreased by code lines removed, should not be a problem.
After this is merged I can look into increasing the over-all coverage.

File: tests/AuthProvider/TokenTest.php [hunk 1]
```diff
@@ -19,23 +19,21 @@ class TokenTest extends TestCase
 {
     public function testCreatingTokenAuthProvider()
     {
-        $options = $this->getOptions();
+        $options                     = $this->getOptions();
```
Comment (edamov) [inline tests/AuthProvider/TokenTest.php]: I didn't find such spaces as part of PSR rules.
Could you please remove these spaces?

**Review by edamov — CHANGES_REQUESTED** [review]
Everything is very good, except for additional spaces in tests

File: tests/AuthProvider/TokenTest.php (hunk 1 above)
Comment (rasmusbe) [reply tests/AuthProvider/TokenTest.php]: Done, it was my editor aligning the `=`
But it's not in the PSR-2 so I removed it.

File: tests/AuthProvider/TokenTest.php [hunk 2]
```diff
@@ -19,9 +19,9 @@ class TokenTest extends TestCase
...
```
Comment (edamov) [inline tests/AuthProvider/TokenTest.php]: Spaces are here also

File: tests/AuthProvider/TokenTest.php [hunk 3]
```diff
@@ -19,9 +19,9 @@ class TokenTest extends TestCase
...
         $options['private_key_path'] = __DIR__ . '/../files/private_key.p8';
-        $authProvider = AuthProvider\Token::create($options);
+        $authProvider                = AuthProvider\Token::create($options);
```
Comment (edamov) [inline tests/AuthProvider/TokenTest.php]: Spaces are here also

File: tests/AuthProvider/TokenTest.php [hunk 4]
```diff
@@ -54,7 +54,7 @@ public function testUseExistingToken()
         $token = 'eyJhbGciOiJFUzI1NiIsImtpZCI6IjEyMzQ1Njc4OTAifQ.eyJpc3MiOiIxMjM0NTY3ODkwIiwiaWF0IjoxNDc4NTE0NDk4fQ.' .
             'YxR8Hw--Hp8YH8RF2QDiwOjmGhTC_7g2DpbnzKQZ8Sj20-q12LrLrAMafcuxf97CTHl9hCVere0vYrzcLmGV-A';
 
-        $options = $this->getOptions();
+        $options      = $this->getOptions();
```
Comment (edamov) [inline tests/AuthProvider/TokenTest.php]: Spaces are here also

File: tests/AuthProvider/TokenTest.php [hunk 5]
```diff
@@ -64,8 +64,8 @@ public function testUseExistingToken()
     private function getOptions()
     {
         return [
-            'key_id' => '1234567890',
-            'team_id' => '1234567890',
+            'key_id'        => '1234567890',
+            'team_id'       => '1234567890',
```
Comment (edamov) [inline tests/AuthProvider/TokenTest.php]: Spaces for these two lines also

**Review by edamov — CHANGES_REQUESTED** [review]

File: tests/AuthProvider/TokenTest.php (hunk 2 above)
Comment (rasmusbe) [reply tests/AuthProvider/TokenTest.php]: I guess it's a matter of preference but since it's your repo you decide 😃 
I have disabled alignment now.

**Review by edamov — APPROVED** [review]

File: tests/AuthProvider/TokenTest.php (hunk 2 above)
Comment (edamov) [reply tests/AuthProvider/TokenTest.php]: Thanks :+1: 
I also used such spaces some time ago.
But I found it didn't follow PSR standarts and didn't add readability in some cases like:
```php
$a                             = 1;
$bVeryLoooongNameOfTheVariable = 2;
$c                             = 3;
$d                             = 4;
```

**Comment by edamov** [comment]
@rasmusbe Thank you! It was the great PR :tada:

**Pull request merged**